tests/test_pretrained_models_download.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+WARNING: Running this test will download ALL pre-trained NeMo models.
+This is bandwidth and disk space consuming.
+"""
+
+import nemo.collections.asr as nemo_asr
+import nemo.collections.nlp as nemo_nlp
+import nemo.collections.tts as nemo_tts
+
+for refresh_cache in [True, False]:
+    # Test ASR collection
+    print(nemo_asr.models.EncDecCTCModel.list_available_models())
+    for model_name in [
```

Comment:
Instead of explicit lists, cant we iterate and fill the list using the `list_available_models()` method and extract the names from there? Its future proof too then.

Reply:
see, my comment above. This is actually testing keys copy-pasted from NGC page.
I'll add a check to make sure that those keys == keys from list_available_models()

Reply:
testing that too